canonical-web-and-design/ubuntu.com
# CUBE: Configure production mappings

## Done
Create a mappings file for production, and keep one for QA and Dev.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts



## Issue / Card
